### DISCUSSION TITLE
I get this error when I run npm start

### DISCUSSION THREAD

**Discussion by EliasH-NMBU** [NASA-SW-VnV/fret#74] [General]
[67656:0207/142004.066829:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!

Does anyone know why this error happens or how to fix it?

**Comment by andreaskatis** [comment]
Hi @EliasH-NMBU,

Since this seems to be an issue with the tool, I created one here:

https://github.com/NASA-SW-VnV/fret/issues/75

Please use the link above to discuss this further.

Best Regards,

Andreas
